### ISSUE TITLE
Revision should point to the current version of the plugin. Within actions this is easy to get, within workers I don't think this is possible, instead we could probably use the version id?

### ISSUE THREAD

**Issue by gentlementlegen** [ubiquity-os/ubiquity-os-logger#46]
Revision should point to the current version of the plugin. Within actions this is easy to get, within workers I don't think this is possible, instead we could probably use the version id?
<img width="461" alt="image" src="https://github.com/user-attachments/assets/ab21ae9e-795d-4fa2-afdf-da776f92f120">

_Originally posted by @gentlementlegen in https://github.com/ubiquity-os/ubiquity-os-kernel/pull/169#discussion_r1818077183_

**Comment by Keyrxng** [comment]
@gentlementlegen we gonna built it into the logger or keep it within the kernel?

**Comment by gentlementlegen** [comment]
@Keyrxng The SDK itself uses the logger but maybe having this metadata in the logger pollutes the code base too much as it is specific to GitHub comments probably.

**Comment by Keyrxng** [comment]
I thought something similar, so best to keep it all within the SDK in error metadata. Can close this or transfer it or whatever as it's resolved within the SDK following the merge of https://github.com/gentlementlegen/ubiquity-os-kernel/pull/1 (will PR the SDK repo now that there is one)

**Comment by gentlementlegen** [comment]
@Keyrxng should we reopen the PR then?

**Comment by Keyrxng** [comment]
I'm including all of the logic for this directly into the SDK right? If so then yeah I'll PR the SDK.

Just seen foo's [comment](https://github.com/ubiquity-os/ubiquity-os-kernel/pull/186?notification_referrer_id=NT_kwDOBlYP6rUxMzE3OTQxNDI0NToxMDYzMDM0NjY#issuecomment-2466685830) re: SDK separation complicating things. Waiting to see if the SDK moves back into the kernel or stays separate

**Comment by gentlementlegen** [comment]
@Keyrxng Also goes for the PR you just closed I guess.

**Comment by Keyrxng** [comment]
> @Keyrxng Also goes for the PR you just closed I guess.

Yes unfortunately, would have been ideal if had managed to merge into your improvements PR like you suggested but it's all good I'll just re-do them.

I assume the SDK is 100% staying separate from the kernel then yeah and I should PR against it?

**Comment by gentlementlegen** [comment]
Let's see how we do it. If the circular reference can be fixed it can remain inside the kernel and it's fine.

**Comment by gentlementlegen** [comment]
Moving this to `plugin-sdk` as it handles what gets inside the `metadata` when `postComment` is called.
